### ISSUE TITLE
App freezes when executing commands

### ISSUE THREAD

**Issue by loopier** [loopier/animatron-godot3#53]
When running routines on groups, it freezes the app on every iteration until all the commands in the routine have been sent. Not sure why this happens, the routine method calls `main.evalCommandList()`. 
```
/load cricle-hd
/create/group c circle-hd 10
/routine bla inf 0.5 /rand/pos c*
```

**Comment by loopier** [comment]
I think the problem is not the routine, but rather the `main.evalCommandList()` method. It freezes the app when any transformation command is called (hardly noticeble when run on only 1 actor). I tried with `/scale` and got the same result.

This is a severe performance issue that can affect heavily the user's experience.

**Comment by loopier** [comment]
I think what's causing the trouble is the OSC command logging. It takes time and when there are a lot of commands to be logged it freezes the app. Changing the `/log/level` to `warn` appears to solve the issue.

**Comment by loopier** [comment]
Fixed in 3e09bc05b3b6c328a23b5d187b52eccbb3d062ca

**Comment by loopier** [comment]
Fixed in 3e09bc05b3b6c328a23b5d187b52eccbb3d062ca
